### DISCUSSION TITLE
Proposal to add UV as dependency manager option

### DISCUSSION THREAD

**Discussion by MiguelLopezVirues** [drivendataorg/cookiecutter-data-science#403] [Requested tools]
Hi team! Great work with this template! As a newbie it is very educational to find CCDS and the good practices proposed in the blog.

I would like to propose the addition of UV by Astral as an option for dependecy management. It is lightning fast compared to Conda, Pipenv and others. I lack the technical expertise to propose the Pull Request myself, but I thought it would be worth to discuss the idea :)

Thanks again!

**Comment by chrisjkuch** [comment]
Support for uv was merged in #408 and will be released in ccds 2.1.0 later this week.

If you don't want to wait, you can use the most recent version of the template by pulling from the master branch, `ccds -c master`

**Comment by chrisjkuch** [comment]
[v2.1.0 released](https://github.com/drivendataorg/cookiecutter-data-science/releases/tag/v2.1.0)!
